packages/lib/src/toolbar/controls/Histogram/Histogram.tsx

```diff
@@ -42,15 +50,15 @@ function Histogram(props: Props) {
   const { width, height } = size;
 
   const xScale = H5WEB_SCALES[scaleType].createScale({
-    domain: xDomain,
+    domain: xDomain && extendDomain(xDomain, EXTEND_FACTOR, scaleType),
```

Comment:
As in `DomainSlider, I now extend the xDomain to have space to move the indicators.